lookout/core/analyzer.py

```diff
@@ -160,6 +160,10 @@ def __init__(self, model: AnalyzerModel, url: str, config: Mapping[str, Any]):
         :param url: The analyzed project's Git remote.
         :param config: Configuration of the analyzer of unspecified structure.
         """
+        for attr_name in ("version", "model_type", "name", "vendor"):
+            if getattr(self, attr_name) is None:
```

Comment:
`getattr(self, attr_name)` raises an AttributeError if no attribute is present. OK?

Reply:
We have all these arguments is set to `None` (https://github.com/src-d/lookout-sdk-ml/pull/79/files#diff-1d5639c92b3eba213af016871b0e26d2R153). Just join 4 separate checks for them into the loop.